sir_cake/store/views.py

```diff
@@ -48,9 +49,45 @@ def store(request):
         if redirect_ is not None:
             return redirect(redirect_)
 
+    context['items_in_basket'] = _get_preorder_or_none(request)
```

Comment:
I suggest using a context processor for this: https://github.com/malisara/planty/search?q=context+processor. Also, I would rename the context key to something like `use_has_items_in_basket`. You can then pass in a boolean, e.g. `_get_preorder_or_none(...) is not None`